pkg/event/event_test.go

```diff
@@ -35,8 +35,8 @@ func monkeyPatch() {
 				CurrentUtcOffsetValid: false,
```

Comment:
Is this the equivalent to UTC_REASONABLE?

Reply:
set CurrentUtcOffsetValid  to true for 6 and 7